fastlane/lib/fastlane/actions/verify_build.rb

```diff
@@ -21,14 +21,18 @@ def self.run(params)
       end
 
       def self.app_path(params, dir)
-        ipa_path = params[:ipa_path] || Actions.lane_context[SharedValues::IPA_OUTPUT_PATH] || ''
-        UI.user_error!("Unable to find ipa file '#{ipa_path}'.") unless File.exist?(ipa_path)
-        ipa_path = File.expand_path(ipa_path)
+        build_path = params[:ipa_path] || params[:build_path] || Actions.lane_context[SharedValues::IPA_OUTPUT_PATH] || ''
+        UI.user_error!("Unable to find file '#{build_path}'") unless File.exist?(build_path)
+        build_path = File.expand_path(build_path)
 
-        `unzip '#{ipa_path}' -d #{dir}`
-        UI.user_error!("Unable to unzip ipa") unless $? == 0
+        if File.extname(build_path) =~ /\.(ipa|zip)/
+          `unzip #{build_path} -d #{dir} -x '__MACOSX/*' '*.DS_Store'`
```

Comment:
Can we re-add single quotes around `#{build_path}`? This breaks if there is a space in the name/path